includes/admin/discounts/class-discount-codes-table.php

```diff
@@ -238,9 +238,21 @@ public function column_name( $discount ) {
 			'edd-action' => 'edit_discount',
 			'discount'   => $discount->id,
 		), $base ) . '">' . stripslashes( $discount->name ) . '</a>' . esc_html( $state ) . '</strong>';
+		
+		/**
+		 * Filter to allow additional content to be appended to the discount title.
+		 *
+		 * @since 3.0
+		 *
+		 * @param EDD_Discount $discount Discount object.
+		 * @param string $base The base URL for the discount list table.
+		 * @param string $state The discount status label.
+		 * @return string Additional data shown in the Name column
+		 */
+		$additional_content = apply_filters( 'edd_discount_row_after_title', '', $discount, $base, $state );
```

Comment:
I know `$state` came from @cklosowski but I'm wondering if this should be the `$status` variable instead ... the `$state` is less easy to evaluate, I think, because it's been modified.  If that's solid, though, then I'm good--just wanting to check first.

Reply:
@robincornett While `$state` could be modified, `$status` may not be the actual `$discount->status`, as you can see we have `$this->get_status()` which is a method of our list table and not the actual `status` of the `EDD_Discount` class, and therefore `$status` is purely used for the column displays.

`$state` is primarily defined as something more accurate than `$status`. Or at least `$discount->status`

Reply:
My concern is that `$state` could be extremely different since it's a "user facing" string, with the em dash and then pulling a discount label (which could be translated, capitalized, etc.).

Reply:
I switched it back to status, but I am fine with either

Reply:
So thinking about this, if we are passing in `$discount` as the `EDD_Discount`, do we even need the `display` version of `$status` that we are getting from `$this->get_status()`? Seems like it would be confusing and people could end up using the wrong `status` indicator. We should revert to a single use of the values when possible and avoid confusion.

Reply:
That seems the best to me for my use case, yeah!

Reply:
@cklosowski the PR currently includes the parameters listed above and I've approved it. If it looks all right to you as well, I can merge once you've approved. Thanks!

Reply:
Since `$status` is not the status of the discount, but the requested status filter, the docblock needs to be updated as it currently reads:
`* @param string $status The discount status.`

Since it is _not_ the discount status, but instead the status requested/filtered by, we need to clarify that.